.github/workflows/ci.yml

```diff
@@ -161,6 +161,44 @@ jobs:
     - name: Check that introspected API and its docs have not diverged
       run: make -C ../doc check
 
+  rust_ci:
+    runs-on: ubuntu-latest
+    env:
+      COVERAGE: 1
+
+    defaults:
+      run:
+        working-directory: ./rust
+
+    strategy:
+      fail-fast: false
+
+    container:
+      image: registry.opensuse.org/opensuse/tumbleweed:latest
+
+    steps:
+
+    - name: Git Checkout
+      uses: actions/checkout@v3
+
+    - name: Configure and refresh repositories
+      # disable unused repositories to have faster refresh
+      run: zypper modifyrepo -d repo-non-oss repo-openh264 repo-update && ( zypper ref || zypper ref || zypper ref )
```

Comment:
Please comment which monster from another dimension is invoked by speaking `zypper ref` three times ! ! !

Reply:
OK, I see it is copied from other jobs, but still it would be nice to know. I prefer Cth*lhu to Candym*an. @lslezak ?

Reply:
I would say that @lslezak do it to prevent ref failures